### ISSUE TITLE
no cosmology

### ISSUE THREAD

**Issue by zingale** [abatten/fruitbat#32]
When I execute the "Custom Cosmologies" part of these docs:
https://fruitbat.readthedocs.io/en/latest/user_guide/using_fruitbat.html

I get:
```
params = {"H0": 72.4, "Om0": 0.26}                                                           

In [38]: new_cosmology = fruitbat.cosmology.create_cosmology(parameters=params)                       
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-38-f71389f8752f> in <module>
----> 1 new_cosmology = fruitbat.cosmology.create_cosmology(parameters=params)

AttributeError: module 'fruitbat' has no attribute 'cosmology'

```

**Comment by abatten** [comment]
This is a typo in the documentation. It should be `fruitbat.cosmologies`.

**Comment by abatten** [comment]
This has been fixed in #33.
